include/swift/SIL/DebugUtils.h

```diff
@@ -237,24 +237,26 @@ bool hasNonTrivialNonDebugTransitiveUsers(
 /// operators to access functionality from the underlying instruction when
 /// needed.
 struct DebugVarCarryingInst {
-  enum class Kind {
+  enum class Kind : uint8_t {
     Invalid = 0,
     DebugValue,
     AllocStack,
     AllocBox,
   };
 
-  Kind kind;
   SILInstruction *inst;
+  Kind kind;
+  uintptr_t spareBits : (sizeof(uintptr_t) - sizeof(Kind)) << 3;
```

Comment:
@atrick this is the bit that I was a little nervous about and that I msged you about. This is just a utility so it /feels/ like it could be ok (and lets me avoid needing to waste more space)... but it still makes me a little uncomfortable.